### PR TITLE
Storybook: Add 'Jetpack Dashboard' background color

### PR DESCRIPTION
**Pull request by retrofox**
This PR adds the Jetpack Dashboard background color, along with Light and Dark ones.

#### Changes proposed in this Pull Request:

* Storybook: Add 'Jetpack Dashboard' background color

#### Testing instructions:

* Run storybook

```bash
cd ./jetpack/projects/js-packages/storybook
```

```bash
> pnpm install & pnpm storybook:dev
```

* Open http://localhost:50240
* Take a look at the background colors
Screen Shot 2022-02-01 at 12 41 35 PM

### CODE REVIEW

**Review by renatoagds — CHANGES_REQUESTED** [review]

File: projects/js-packages/storybook/storybook/preview.js [hunk 1]
```diff
@@ -17,3 +17,23 @@ import { withA11y } from '@storybook/addon-a11y';
...
 
 addDecorator( withA11y );
+
+export const parameters = {
+	backgrounds: {
+		default: 'Jetpack Dashboard',
+		values: [
+			{
+				name: 'Jetpack Dashboard',
+				value: '#F6F6F6',
```
[the remark this answers is not in the document]
Comment (retrofox) [reply projects/js-packages/storybook/storybook/preview.js:27]: so these styles are already exposed in the storybook context?

[the remark this answers is not in the document]
Comment (retrofox) [reply projects/js-packages/storybook/storybook/preview.js]: Addressed here 46e28d8f675ca0582814b10096173c9b800f3185

[the remark this answers is not in the document]
Comment (renatoagds) [reply projects/js-packages/storybook/storybook/preview.js]: The hexadecimal value is exposed as css variables. I don't know if we can use it in `parameters`.

[the remark this answers is not in the document]
Comment (renatoagds) [reply projects/js-packages/storybook/storybook/preview.js]: Saw that you already used it

**Review by renatoagds — DISMISSED** [review]
LGTM

Screen Shot 2022-02-01 at 14 11 08

**Review by renatoagds — APPROVED** [review]

**Pull request merged**